xbmc/music/Album.cpp

```diff
@@ -99,11 +98,11 @@ CAlbum::CAlbum(const CFileItem& item)
   { // no musicbrainz info, so fill in directly
     for (std::vector<std::string>::const_iterator it = tag.GetAlbumArtist().begin(); it != tag.GetAlbumArtist().end(); ++it)
     {
-      std::string strJoinPhrase = (it == --tag.GetAlbumArtist().end() ? "" : g_advancedSettings.m_musicItemSeparator);
-      CArtistCredit artistCredit(*it, "", strJoinPhrase);
+      CArtistCredit artistCredit(*it);
       artistCredits.push_back(artistCredit);
```

Comment:
merge these two lines and use artistCredits.emplace_back(*it); for some "bonus performance", it avoids some copying.